ocs_ci/deployment/image_registry.py

```
@@ -0,0 +1,405 @@
+        """
+        logger.info(f"Waiting for PVC {self.pvc_name} to be Bound")
+        ocp = OCP(kind="pvc", namespace=self.namespace, resource_name=self.pvc_name)
+
+        try:
+            ocp.wait_for_resource(
+                condition="Bound",
+                resource_name=self.pvc_name,
+                timeout=timeout,
+                sleep=10,
+            )
+            logger.info(f"PVC {self.pvc_name} is now Bound")
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)